cpp/src/IceBox/Admin.cpp

```diff
@@ -67,18 +98,18 @@ Client::run(int argc, char* argv[])
     vector<string> commands;
     try
     {
-        commands = opts.parse(argc, const_cast<const char**>(argv));
+        commands = opts.parse(args);
```

Comment:
I think it would be more logical to put the result in args. Otherwise, pass args as a const&.